src/pages/NewChairConsole.js

```diff
@@ -107,11 +107,10 @@ const Records = (props) => {
         case 'Open' :
           const votes = filter({type: 'DAC', dacUserId: currentUserId})(electionInfo.votes);
           const isFinal = !isEmpty(votes.find((voteData) => !isNil(voteData.vote)));
-          const vote = head(votes);
```

Comment:
You can keep this line here, I'm going to need to know if a vote exists for the disabled button styling

Reply:
Okay, I wonder if we will still need the disabled buttons? Maybe it was only erroring when there were no votes bc it was trying to put the voteid on the end of the link

Reply:
Actually scratch that, I can just check if the array is empty

Reply:
@JVThomas - I don't see the usage of `vote` in this method, are you planning on it being there for another PR?

Reply:
The disabled buttons still need to be there because we need to let the user know that the button won't work for them. We shouldn't have to let the figure it out.

Reply:
The vote isn't needed, just the array, hence the "scratch that" comment. The array is needed for my PR, which is going to add disabled styling to the vote button if the user doesn't have a vote